packages/react/src/feature-app-container.tsx

```diff
@@ -55,14 +55,23 @@ export interface FeatureAppContainerProps {
 type InternalFeatureAppContainerProps = FeatureAppContainerProps &
   FeatureHubContextValue;
 
+interface InternalFeatureAppContainerState {
+  hasFeatureAppError: boolean;
```

Comment:
Please add a comment regarding the purpose of use

Reply:
Will be set to true when a DOM Feature App throws in #attachTo or when the error boundary catches an error thrown by a React Feature app in a lifecycle method. Since DOM Feature Apps aren't server-side-rendered and error boundaries only work on the client, this flag will only ever be true on the client.